src/main/java/com/kylenicholls/stash/parameterizedbuilds/item/Server.java

```diff
@@ -8,19 +8,26 @@
 	private String user;
 	private String token;
 	private boolean altUrl;
+	private boolean csrfEnabled;
 
-	public Server(String baseUrl, String user, String token, boolean altUrl) {
+	public Server(String baseUrl, String user, String token, boolean altUrl, boolean csrfEnabled) {
 		this.baseUrl = baseUrl;
 		this.user = user;
 		this.token = token;
 		this.altUrl = altUrl;
+		this.csrfEnabled = csrfEnabled;
 	}
 
 	public Server(Map<String, Object> map) {
 		this.baseUrl = (String) map.get("baseUrl");
 		this.user = (String) map.get("user");
 		this.token = (String) map.get("token");
 		this.altUrl = Boolean.parseBoolean(map.get("altUrl").toString());
+		try {
+			this.csrfEnabled = Boolean.parseBoolean(map.get("csrfEnabled").toString());
```

Comment:
Instead of doing the try catch block, I think we could just use `map.getOrDefault("csrfEnabled", true)` to supply the default value.

Reply:
some functions I don't know, I don't code a lot, I was teached on java 1.2 ;)